scripts/run_autonomous.sh

```
@@ -223,6 +223,19 @@ $prompt_content
     log_info "Returning to develop branch..."
     git checkout develop 2>/dev/null || log_warn "Could not checkout develop — may already be there"
 
+    # Merge the previous prompt's branch INTO develop locally (no PR merge)
+    # This lets the next prompt see prior work, while PRs stay open for review
+    local prev_branch=$(git log --all --format='%D' -20 | grep -o 'feature/[^ ,]*' | head -1)
```

Shellcheck (0.11.0)

[warning] Declare and assign separately to avoid masking return values.

(SC2155)